src/components/Episodes/Episodes.jsx

```diff
@@ -12,16 +13,18 @@ export default function Episodes(){
 
 	return (
 		<article>
-			<ol>
-				{ episodes }
-			</ol>
-			<nav>
-				<button aria-label="Previous episode." />
-				<button aria-label="Next episode." />
+			<EpisodesProvider>
 				<ol>
-					{ pagination }
+					{ episodes }
 				</ol>
-			</nav>
+				<nav>
+					<button aria-label="Previous episode." />
+					<button aria-label="Next episode." />
```

Comment:
might be worth putting `aria-controls` on here and pointing to the episodes' `<ol>`

Reply:
Done. Used "episodes__items" as list id (see changes in nav branch).